src/Expander/Error.hs

```diff
@@ -34,7 +34,7 @@ data ExpansionErr
   | NotInteger Syntax
   | NotString Syntax
   | NotModName Syntax
-  | NotRightLength Natural Syntax
+  | NotRightLength [Natural] Syntax
```

Comment:
Generalizing the "expected a list of length 3" message to "expected a list of length 1, 2, or 3".